### ISSUE TITLE
JaxOpt is no longer maintained

### ISSUE THREAD

**Issue by Zethson** [ott-jax/ott#653]
```
../../miniforge3/envs/pertpy/lib/python3.12/site-packages/jaxopt/__init__.py:59
  python3.12/site-packages/jaxopt/__init__.py:59: DeprecationWarning: JAXopt is no longer maintained. See https://docs.jax.dev/en/latest/ for alternatives.
    warnings.warn(
```

JaxOpt is a dependency of ott-jax and should ideally be replaced with something else.

**Comment by marcocuturi** [comment]
this is a good point. We're mostly using `Jaxopt` to run proximal gradient descent and some `h`-transforms. Maybe we should copy-paste their code as they haven't moved what we're interested in in optax.
